flakes.yaml

```
@@ -6,3 +6,8 @@
 # "test/new-e2e/tests/agent-platform/install-script"
 #   - "TestInstallScript/test_install_script_on_centos-79_x86_64_datadog-agent_agent_7"
 
+# TODO: https://datadoghq.atlassian.net/browse/CONTINT-4143
+test/new-e2e/tests/containers:  
+  - TestEKSSuite/TestCPU/metric___container.cpu.usage{^kube_deployment:stress-ng$,^kube_namespace:workload-cpustress$}
+  - TestEKSSuite/TestCPU
+  - TestEKSSuite
```

I would have two questions:

* Is it needed to recursively list all the parents of a flaky test ? Ideally, I’d love to list only the “real flaky” one `TestEKSSuite/TestCPU/metric___container.cpu.usage{^kube_deployment:stress-ng$,^kube_namespace:workload-cpustress$}` and not all its parents.
  If we accept that a failing sub-test makes its parents fail, should that be taken into account by the framework reading `flakes.yaml` ?

* Does `flakes.yaml` support patterns ?
  `TestCPU` is executed inside two suites: `TestEKSSuite` and `TestKindSuite`.

```suggestion
test/new-e2e/tests/containers:  
  - Test(EKS|Kind)Suite/TestCPU/metric___container.cpu.usage{^kube_deployment:stress-ng$,^kube_namespace:workload-cpustress$}
```